index.html

```diff
@@ -0,0 +1,874 @@
+        <p class="text-center text-white"><span class="vs p-2">Vs</span></p>
+        <hr class="debate-con mb-0">
+        <img src="img/foundation.jpg" alt="Image Debate" class="float-left mr-4">
+        <h2 class="h6 mt-4 mb-5 font-weight-bold">Bootstrap is not the framework of the future</h2>
+        <img src="img/bootstrap.webp" alt="Image Profile" class="rounded-circle float-right">
+        <hr class="mb-1 debate">
+        <hr class="m-0 debate">
+        <p class="text-uppercase">By <span class="font-weight-bold">Zurb Foundation</span></p>
+      </article>
+      <!-- Sponsored Insights -->
+      <section class="sponsored flex-column bg-white mt-6">
+        <h5 class="d-none">Sponsored Insight</h5>
+          <article class="art-container">
+          <figure class="img-container m-auto">
+            <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+            <figcaption>
+              <h3 class="text-uppercase text-primary bottom-left p-2 h6 font-weight-bold">Sponsored Insight</h3>
+            </figcaption>
+          </figure>
+          <div class="border">
+            <h3 class="font-weight-bold">Winter is coming!</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          </div>
+        </article>
+          <article class="art-container">
+            <figure class="img-container m-auto">
+              <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+              <figcaption>
+                <h3 class="text-uppercase text-primary bottom-left p-2 h6 font-weight-bold">Sponsored Insight</h3>
+              </figcaption>
+            </figure>
+            <div class="border">
+              <h3 class="font-weight-bold">Winter is coming!</h3>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </article>
+          <article class="art-container">
+            <figure class="img-container m-auto">
+              <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+              <figcaption>
+                <h3 class="text-uppercase text-primary bottom-left p-2 h6 font-weight-bold">Sponsored Insight</h3>
+              </figcaption>
+            </figure>
+            <div class="border">
+              <h3 class="font-weight-bold">Winter is coming!</h3>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </article>
+      </section>
+    <!-- More stories -->
+      <article class="more-stories bg-white">
+        <h5 class="text-muted text-uppercase">More stories</h5>
+        <section class="border-top">
+          <img src="img/bootstrap.png" alt="Image Debate" class="float-left mr-4">
```

Comment:
- [ ] The articles in this section have a larger image when you are in the medium breakpoint and below. Please adjust them.